gtest/src/manager.rs

```diff
@@ -146,6 +173,7 @@ pub(crate) struct ExtManager {
     // State
     pub(crate) actors: BTreeMap<ProgramId, (Actor, Balance)>,
     pub(crate) codes: BTreeMap<CodeId, Vec<u8>>,
```

Comment:
this might be renamed or some comments left, that actual `codes` is exactly `opt_binaries`